PlayCover/Views/Settings/IPASourceSettings.swift

```diff
@@ -11,23 +11,27 @@ struct SourceData: Identifiable, Hashable {
     var id = UUID()
     var source: String
     var status: SourceValidation = .valid
+    var isEnabled: Bool
 
     enum SourceDataKeys: String, CodingKey {
         case source
+        case isEnabled
     }
 }
 
 extension SourceData: Encodable {
```

Comment:
Can you change the way `SourceData` encodes and decodes data to be the same as keymappings does it ([Model/Keymapping.swift](https://github.com/PlayCover/PlayCover/blob/develop/PlayCover/Model/Keymapping.swift))? This would make it easier to read without needing the use of `CodingKeys` while also preserving types.